ui/lib/pki/addon/templates/overview.hbs

```diff
@@ -19,7 +19,49 @@
 </Toolbar>
 
 {{#if this.model.hasConfig}}
-  {{! TODO show overview items }}
+  <div
+    class="selectable-card-container has-grid has-top-margin-l has-two-col-grid
+      {{if (or (eq this.model.issuers 403) (eq this.model.roles 403)) 'has-three-col-grid' 'has-two-col-grid'}}"
```

Comment:
Did you ask crypto if the issuers endpoint will ever `403`? We might be able to simplify this conditional to 
```suggestion
      {{if this.rolesCanList 'has-three-col-grid' 'has-two-col-grid'}}"
```